Gruntfile.js

```diff
@@ -92,6 +92,7 @@ module.exports = function(grunt) {
   // on each recompile
   var updateTsFiles = function() {
     tsFiles = grunt.file.read("src/reference.ts")
+      .replace(/\r\n/g, "\n")
```

Comment:
Doing some more testing, sed:privateDefinitions also gives unwanted results with windows line ends, and this would be a lot harder to resolve... so fixing up the line ends with .gitattributes seems the most robust solution.